components/mpas-seaice/driver/ice_comp_mct.F

```diff
@@ -1759,9 +1759,8 @@ subroutine ice_import_mct(x2i_i, errorCode)!{{{
       i,n
 
    real (kind=RKIND) :: &
-      frazilProduction, &
-      meltingPotential, &
-      freezingPotential
+      frazilMassFlux, &
+      frazilMassFluxRev
```

Comment:
What does Rev stand for? I would suggest a name describing the variables function

Reply:
Please see the revised documentation.  I set this up as a draft, and it was not ready for review at the time you looked at this.  However, I believe the in-code documentation answers your questions.